.github/workflows/releaser.yml

```diff
@@ -43,12 +43,12 @@ jobs:
             env  BINDGEN_EXTRA_CLANG_ARGS='--sysroot /usr/aarch64-linux-gnu' CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=/usr/bin/aarch64-linux-gnu-gcc cargo build --profile maxperf --target=aarch64-unknown-linux-gnu -p avail-light           
             mv target/aarch64-unknown-linux-gnu/maxperf/avail-light target/aarch64-unknown-linux-gnu/maxperf/avail-light-linux-aarch64
             pushd target/aarch64-unknown-linux-gnu/maxperf/
-            tar czf avail-light-linux-aarch64.tar.gz avail-light-linux-aarch64
+            tar czf avail-light-linux-arm64.tar.gz avail-light-linux-aarch64
             popd
          - uses: actions/upload-artifact@v2
            with:
-             name: avail-light-linux-aarch64-binary
-             path: target/aarch64-unknown-linux-gnu/maxperf/avail-light-linux-aarch64.tar.gz
+             name: avail-light-linux-arm64-binary
```

Comment:
Any chance we might add a Windows build? Only additional effort would be that it would require clang for compile...

Reply:
I'd rather work on this in a separate PR.
Don't wanna couple many changes. I'll put it on the back log for upcoming week.